test/hotspot/jtreg/compiler/loopopts/TestPeelingSkeletonPredicateInitialization.java

```diff
@@ -32,7 +32,7 @@
  *          5. The rangecheck is still before the peeling, and is not copied to the peeled loop. Hence
  *             we do not statically realize that the peeled loop can never be entered.
  *
- * @run main/othervm -XX:+UnlockDiagnosticVMOptions -XX:+StressLCM -XX:+StressGCM -XX:+StressCCP -XX:+StressIGVN
+ * @run main/othervm -XX:+UnlockDiagnosticVMOptions -XX:+StressLCM -XX:+StressGCM -XX:+StressCCP -XX:+StressIGVN -XX:+StressMacroExpansion
```

Comment:
Same here, I would not add this option here since it's a regression test for a specific bug.